Blosc/config.py

```diff
@@ -0,0 +1,24 @@
+{
+
+	"downloads" : [
+
+		"https://github.com/Blosc/c-blosc/archive/v1.14.4.tar.gz"
+
+	],
+
+	"license" : "LICENSES/BLOSC.txt",
+
+	"commands" : [
+
+		"cmake -E make_directory {buildDir}/doc/licenses/blosc && cmake -E copy LICENSES/* {buildDir}/doc/licenses/blosc/",
+		"mkdir gafferBuild",
+		"cd gafferBuild &&"
+			" cmake"
+			" -G \"Unix Makefiles\""
```

Comment:
This surprises me, since it seems anti-Windows. Remind me, is this a WSL thing? And if so, is that the long term plan for building Windows dependencies?

Reply:
Yeah that's anti-windows, I'm planning to roll out windows patches at a later date after these all work on linux. Eric and I did modify build.py quite a bit and I wanted to do that change later.